telemetry_examples/src/main/java/com/newrelic/telemetry/examples/SpanExample.java

```diff
@@ -5,7 +5,7 @@
 package com.newrelic.telemetry.examples;
 
 import com.newrelic.telemetry.Attributes;
-import com.newrelic.telemetry.MetricBatchSenderFactory;
+import com.newrelic.telemetry.SpanBatchSenderFactory;
```

Comment:
This needs to move down two lines per google java formatting.

Reply:
Sorry about that! The imports are now fixed.